third_party/tutorial.proto

```diff
@@ -0,0 +1,17 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package tutorial;
```

Comment:
Why does the file end here?

Reply:
This will be the base proto for `master` and all future branches. I'll act as a dummy for all CI builds. This is a general solution for the proto dependencies on CI, as otherwise we'd need to have a [branching](https://docs.travis-ci.com/user/conditional-builds-stages-jobs/#conditional-stages) or [graceful errors](https://docs.travis-ci.com/user/conditional-builds-stages-jobs/#conditional-stages) (per PR description).

We will move the finalized proto file once the proto rendering + parsing is stable (and back port it to the other repo) once that code is finalized.